docs/examples/driver_examples/Qcodes example with Ithaco.ipynb

```diff
@@ -158,239 +238,561 @@
+       "   'instrument_name': 'camp',\n",
+       "   'name': 'invert',\n",
+       "   'inter_delay': 0,\n",
+       "   'unit': '',\n",
+       "   'vals': '<Boolean>',\n",
+       "   'post_delay': 0,\n",
+       "   'label': 'Inverted output'},\n",
+       "  'sens_factor': {'__class__': 'qcodes.instrument.parameter.Parameter',\n",
+       "   'full_name': 'camp_sens_factor',\n",
+       "   'value': 1,\n",
+       "   'raw_value': 1,\n",
+       "   'ts': '2020-11-06 13:07:06',\n",
+       "   'instrument': 'qcodes.instrument_drivers.ithaco.Ithaco_1211.Ithaco_1211',\n",
+       "   'instrument_name': 'camp',\n",
+       "   'name': 'sens_factor',\n",
+       "   'inter_delay': 0,\n",
+       "   'unit': '',\n",
+       "   'vals': '<Enum: {0.1, 1, 10}>',\n",
+       "   'post_delay': 0,\n",
+       "   'label': 'Sensitivity factor'},\n",
+       "  'suppression': {'__class__': 'qcodes.instrument.parameter.Parameter',\n",
+       "   'full_name': 'camp_suppression',\n",
+       "   'value': 1e-07,\n",
+       "   'raw_value': 1e-07,\n",
+       "   'ts': '2020-11-06 13:07:01',\n",
+       "   'instrument': 'qcodes.instrument_drivers.ithaco.Ithaco_1211.Ithaco_1211',\n",
+       "   'instrument_name': 'camp',\n",
+       "   'name': 'suppression',\n",
+       "   'inter_delay': 0,\n",
+       "   'unit': 'A',\n",
+       "   'vals': '<Enum: {1e-09, 1e-05, 1e-08, 1e-10, 0.0001, 1e-07, 1e-06, 0.001}>',\n",
+       "   'post_delay': 0,\n",
+       "   'label': 'Suppression'},\n",
+       "  'risetime': {'__class__': 'qcodes.instrument.parameter.Parameter',\n",
+       "   'full_name': 'camp_risetime',\n",
+       "   'value': 0.3,\n",
+       "   'raw_value': 0.3,\n",
+       "   'ts': '2020-11-06 13:07:01',\n",
+       "   'instrument': 'qcodes.instrument_drivers.ithaco.Ithaco_1211.Ithaco_1211',\n",
+       "   'instrument_name': 'camp',\n",
+       "   'name': 'risetime',\n",
+       "   'inter_delay': 0,\n",
+       "   'unit': 'msec',\n",
+       "   'vals': '<Enum: {0.03, 1, 0.1, 0.3, 3, 100, 1000, 10, 0.01, 300, 30}>',\n",
+       "   'post_delay': 0,\n",
+       "   'label': 'Rise Time'}},\n",
+       " 'name': 'camp'}"
       ]
      },
-     "execution_count": 16,
+     "execution_count": 11,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "plotQ"
+    "camp.snapshot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up qcodes database and performing a simple measurement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we want to feed our Itacho using our Keithley sourcemeter and run a simple qcodes measurement to see the output of the swept values passing through Itacho (for knowing how our sourcemeter works, you can refer to its qcodes notebook in this link https://qcodes.github.io/Qcodes/examples/driver_examples/Qcodes%20example%20with%20Keithley%202600.html):"
```

Comment:
Please change this link to have a nice text with the title of the notebook. (using markdown style links the documentation of nbsphinx should have an example of how to do this)